evennia/utils/evmore.py

```diff
@@ -111,9 +111,13 @@ class CmdMoreExit(Command):
     def func(self):
         """
         Exit pager and re-fire the failed command.
-
         """
         more = self.caller.ndb._more
+        if not more and inherits_from(self.caller, evennia.DefaultObject):
+            more = self.caller.account.ndb._more
```

Comment:
Should I rewrite the other command that already uses this approach as well, then?

Reply:
If another command uses this bit, I think it would be susceptible to the same traceback. So if you fix that too that would be great! Otherwise make a ticket for it :)

Reply:
Okay, I'll update both!